lib/src/models/chat_view_states_configuration.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:chatview/src/values/typedefs.dart';
+import 'package:flutter/material.dart';
+
+class ChatViewStateConfiguration {
+  const ChatViewStateConfiguration({
+    this.noMessageWidget,
+    this.loadingWidget,
+    this.errorWidget,
+    this.errorWidgetConfig = const ChatViewStateWidgetConfiguration(
+      defaultTitle: 'Error',
+      defaultTitleTextStyle: TextStyle(
+        fontSize: 22,
+      ),
+    ),
+    this.noMessageWidgetConfig = const ChatViewStateWidgetConfiguration(
+      defaultTitle: 'No Messages',
+      defaultTitleTextStyle: TextStyle(fontSize: 22),
+    ),
+    this.loadingWidgetConfig,
+  });
+
+  final Widget? noMessageWidget;
+  final Widget? loadingWidget;
+  final Widget? errorWidget;
+  final ChatViewStateWidgetConfiguration? errorWidgetConfig;
+  final ChatViewStateWidgetConfiguration? noMessageWidgetConfig;
+  final ChatViewStateWidgetConfiguration? loadingWidgetConfig;
+}
+
+class ChatViewStateWidgetConfiguration {
+  const ChatViewStateWidgetConfiguration({
+    this.defaultTitle,
```

Comment:
we can remove default keyword from all params

Reply:
Resolved